tpcx_bb/benchmark_runner/.ipynb_checkpoints/benchmark_config-checkpoint.yaml

```diff
@@ -0,0 +1,6 @@
+# benchmark config yaml
```

Comment:
Can you remove the `.ipynb_checkpoints` files from here please

Reply:
Surprised they aren't in the gitignore, but then I realized we didn't copy it over 😄

Reply:
Ahh, yeah, thats why. Was wondering the same thing.